static/app/views/performance/browser/webVitals/pageOverviewWebVitalsDetailPanel.tsx

```diff
@@ -53,11 +52,23 @@ const columnOrder: GridColumnOrder[] = [
   {key: 'score', width: COL_WIDTH_UNDEFINED, name: 'Score'},
 ];
 
+const inpColumnOrder: GridColumnOrder[] = [
+  {key: 'profile.id', width: COL_WIDTH_UNDEFINED, name: 'Profile'},
+  {key: 'replayId', width: COL_WIDTH_UNDEFINED, name: 'Replay'},
+  {key: 'webVital', width: COL_WIDTH_UNDEFINED, name: 'Inp'},
+  {key: 'score', width: COL_WIDTH_UNDEFINED, name: 'Score'},
```

Comment:
Do the `name` properties render as labels? Then I guess we should probably hit them with `t()`

Reply:
Some of them are used as labels, added localization!